### DISCUSSION TITLE
Has anyone tried using Google Cloud Dataflow yet?

### DISCUSSION THREAD

**Discussion by ElijahKSmith** [planetscale/discussion#641] [Q&A] [answered]
Currently looking to load my PlanetScale data into my BigQuery warehouse and checking the available options, but I was wondering if Dataflow would work and if anyone had tried it yet. If so, what was your experience like?

**Comment by swiecki** [comment]
We're facing the same question trying to duplicate our db to bigquery- datastream is asking for slave replication permission and i'm not sure what planetscale supports

**Reply by jgreet** [reply, marked as answer]
Hi, the answer from https://github.com/planetscale/discussion/discussions/474 is still valid here.

PlanetScale doesn't yet provide binlog access, so you won't be able set up Google Datastream, but it does offer integrations with several ETL platforms:

https://planetscale.com/docs/concepts/planetscale-connect
